docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -264,6 +264,10 @@ The `status` object **may** define several fields:
 * `externalManagedControlPlane` - is a bool that should be set to true if the Node objects do not
   exist in the cluster. For example, managed control plane providers for AKS, EKS, GKE, etc, should
   set this to `true`. Leaving the field undefined is equivalent to setting the value to `false`.
+* `.Conditions[Paused]` - is a condition that reports if the cluster or control plane resource is paused. It should check if 'spec.paused' is set on the cluster, and for the paused annotation on the resource. This is currently optional, but will become required in future.
+  ```go
+{{#include ../../providers/code-examples.md:2:28}}
```

Comment:
I would suggest to not add a code example now, given that most probably we will implement a util to standardize implementation both in CAPI and across providers
(we can add the example in a follow up)